mmp/src/main/java/org/iit/mmp/helper/HelperClass.java

```diff
@@ -11,21 +11,37 @@ public HelperClass(WebDriver driver)
 	{
 		this.driver= driver;    
 	}
+	
 	public void login(String uname,String pword,String url)
 	{
 		launchBrowser(url);
 		driver.findElement(By.id("username")).sendKeys(uname);
 		driver.findElement(By.id("password")).sendKeys(pword);
 		driver.findElement(By.name("submit")).click();
 	}
+	
+	//Overloaded method - RA
+	public void login(String uname, String pwd) {
+		driver.findElement(By.id("username")).sendKeys(uname);
+		driver.findElement(By.id("password")).sendKeys(pwd);
+		driver.findElement(By.name("submit")).click();
+	}
+
 	public void navigateToAModule(String moduleName)
 	{
 		driver.findElement(By.xpath("//span[contains(text(),'"+moduleName+"')]")).click();
 	}
+	
 	public void launchBrowser(String url)
 	{
-		driver.get(url);
-		 
+		driver.get(url);	 
+	}
+	
```

Comment:
Login method developed same can be used for Patient and Admin as well. Remove this method and send the pull request again.

Reply:
Have used different Login methods for Patient and Admin modules because the Sign In button names are different for both modules. For Patient module button name is 'submit' and for Admin module Sign In button name is 'admin'. Please can you review remaining code. Thanks